eng/pipelines/templates/BuildAndTest.yml

```diff
@@ -86,6 +86,7 @@ steps:
     - template: /eng/pipelines/templates/send-to-helix.yml
       parameters:
         HelixProjectPath: '$(Build.SourcesDirectory)/tests/helix/send-to-helix-ci.proj'
+        # Code coverage on windows is disabled: <issue>
```

Comment:
Was this intentional? Was this supposed to have an issue linked?

Reply:
We should remove it, I will remove it in the backport PR if that's fine, I don't want to wait another build for that